java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureGroupBase.java

```diff
@@ -102,6 +102,10 @@ public class FeatureGroupBase {
   @Setter
   protected String location;
 
+  @Getter
+  @Setter
+  protected TimeTravelFormat timeTravelFormat = TimeTravelFormat.HUDI;
```

Comment:
Make sure for external feature groups the backend doesn't complain if this is being sent.

Reply:
the are no complaints, i can create an external feature group